themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_nb_updates.scss

```diff
@@ -22,9 +22,9 @@
   .main.width-container {
     background: transparent;
   }
-  .footer-nav {
-    display: none;
-  }
+  // .footer-nav {
+  //   display: none;
+  // }
```

Comment:
What is removing these `display: none` for?

Reply:
it's unnecessary b/c that file isn't used, so I reverted